apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -325,7 +367,11 @@ class VisualizerModal extends React.Component {
               {isMultiColumnChart && (
                 <DropdownField
                   displayName={msg.dataVisualizerYValues()}
-                  options={this.props.tableColumns}
+                  options={this.filterTableColumnsByCount(
```

Comment:
Can we disable the dropdown if there are no options to choose from?

Reply:
I'm leaving it enabled, but I did flip around the logic so that it'll pop up and show the columns as disabled, potentially up to all of them.